packagingenv/centos8/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM centos:8
+
+RUN yum -y install gcc-c++ make git libicu-devel rpmdevtools
+
+# Add NodeSource repo
+RUN curl --silent --location https://rpm.nodesource.com/setup_10.x | bash -
+
+# Install development tools
+RUN dnf -y module install nodejs:10
```

Comment:
Any specific benefit behind using `dnf`? As I remember `yum` worked well too for installing nodejs.

If we can, ideally to use the same tooling as we'll do for installers for simplicity and consistency reasons.

Reply:
it's the same as yum. But I put it back